docs/source/software/systemwide/matlab.md

```diff
@@ -61,95 +61,372 @@ Use the following procedure if you need to install a MATLAB toolbox:
-
-This walkthrough will use Open OnDemand, the web portal on the cluster, to launch MATLAB. You will then use the default cluster profile.This allows you to utilize the resources that are allocated to your Open On Demand MATLAB job.
-
-To start MATLAB and work with the Parallel Server, do the following:
+### Configuration of MATLAB client on the HPC
+The cluster has MATLAB Parallel Server installed and this section details an example of how you can set up and use the MATLAB Parallel Computing Toolbox on the HPC. This walkthrough uses MATLAB R2023a launched as an interactive app on the [Open OnDemand](ood.discovery.neu.edu) web portal. There are several parts to this walkthrough so we suggest that you read it through completely before starting. The parameters presented represent only one scenario and the results may differ when you run the examples.
 
 1. Go to <http://ood.discovery.neu.edu>. If prompted, sign in with your cluster username and password.
 
 1. Click **Interactive Apps**, and select **MATLAB**.
 
-1. Select **MATLAB version 2023a**, and set the time for the length of your jobs, the number of cpus to be 1 plus the number you plan to use for the parallel toolbox work, and default memory to at least 2 times the number of cpus you request. Click **Launch**.
+1. Select **MATLAB version R2023a** or newer, and set the time for the length of your job, the number of cpus to be 8, and the memory to at least 16 GB. Click **Launch**.
 
 1. If necessary, adjust the **Compression** and **Image Quality**, and then click **Launch MATLAB**.
 
-The following commands `parpool(feature('numcores'))` will start the Parallel Server with the number of cores that are allocated in the MATLAB Open OnDemand job (`feature('numcores')`). You can inspect the Parallel Server and the state of it with the command `parcluster`.
+1. Once MATLAB is open, click on the Home tab, click Parallel > Discover Clusters... to discover the profile. Note, this is valid for R2023a and newer. Alternatively, you can create a cluster profile to run parallel jobs by running `configCluster` in the Command Window. Note: `configCluster` should only be called once on the HPC.
 
-:::{code-block} matlab
-#with parpool
-parpool(feature('numcores'))
+Now MATLAB is configured to have jobs submitted to the HPC cluster and not run in the current session.
 
-#with parcluster
-parcluster
-:::
+### Installation and Configuration of MATLAB on a Local Machine
+MATLAB can be configured on your local machine to submit jobs to the HPC cluster. To complete this, you will need to download the following directory from Github: [MATLAB Desktop Parallel Toolbox Setup](https://github.com/northeastern-rc/matlab-configuration-scripts).
 
-### Using parcluster example
+In your desktop MATLAB instance, run:
```

Comment:
the instructions go back and forth between desktop and HPC. I wonder if we group them in their own sections so no one loses track that they are reading desktop instructions?

Reply:
I broke this out to be tabs. You are right that this was a strange back and forth.